feature/home/src/main/res/values/strings.xml

```diff
@@ -28,6 +28,7 @@
     <string name="customize_your_own">Customize your own\n</string>
     <string name="into_an_android_bot">\n\nAndroid bot</string>
     <string name="home_button_label">Let\'s Go</string>
+    <string name="full_space_button_label">To Full Space</string>
```

Comment:
can you prefix or suffix with xr? The wording of "full space" isn't entirely clear that its XR related.